src/stories/Components/Timeline/AssignmentDutyCard.stories.tsx

```diff
@@ -13,7 +13,7 @@ const blank_duty: AssignmentDuty = {
     id: '0',
```

Comment:
Would be nice if we could get the stories working again sometime.  Perhaps if we ever choose to move some of the layout logic up a level, we can get these working again.